packages/zoe/src/contractFacet/zcfZygote.js

```diff
@@ -340,7 +340,15 @@ export const makeZCFZygote = (
    * @type {ZCFZygote}
    * */
   const zcfZygote = {
-    evaluateContract: bundle => {
+    evaluateContract: bundleOrBundlecap => {
+      let bundle;
+      if (passStyleOf(bundleOrBundlecap) === 'remotable') {
+        const bundlecap = bundleOrBundlecap;
+        // @ts-ignore powers is not typed correctly: https://github.com/Agoric/agoric-sdk/issues/3239s
```

Comment:
@turadg recommends that we prefer `@ts-expect-error` when that works as well. That way, if it stops being necessary, it'll let us know. Also, typo
```suggestion
        // @ts-ignore powers is not typed correctly: https://github.com/Agoric/agoric-sdk/issues/3239
```

Reply:
👍, modulo that due to https://github.com/Agoric/agoric-sdk/issues/4560 sometimes the IDE and CLI/CI will disagree about the set of errors. In those cases it's better to use ts-ignore than to have either of them complain. After that issue is resolved we could global replace ts-ignore with ts-expect-error and see which are gone.

In this particular case I think we're better off fixing the types: 1b043d284c13aa3679db05d6f1be42b43a2bafc3 I didn't push directly to this branch because I didn't know the norms here around that and I wasn't certain I got it right.

Reply:
>  I didn't push directly to this branch because I didn't know the norms here around that and I wasn't certain I got it right.

I defer to @warner on such norms. But my take:

Don't push directly to someone else's branch. Create a PR based on their's, send it to them to review, and mention that PR in a review comment on their original PR. If they like it, they should feel free to merge it into their PR without further consulting you.

Reply:
yeah, either make a meta-PR, or use github's "suggested change" mechanism.. pushing to my branch would surprise/confuse my local workspace and/or workflow

Reply:
👌 no surprise pushing to PR branches

for something that's too small to make a new PR but too spread out to make a bunch of suggested changes, WDYT of sharing the commit as above? 1b043d284c13aa3679db05d6f1be42b43a2bafc3  Then you can just `git cherry-pick 1b043d2` if you want it.

Reply:
@warner I notice TS failures in https://github.com/Agoric/agoric-sdk/runs/5300164567?check_suite_focus=true

If that's due to https://github.com/Agoric/agoric-sdk/commit/1b043d284c13aa3679db05d6f1be42b43a2bafc3 no objections here to punting that.